stdpopsim/generic_models.py

```diff
@@ -0,0 +1,87 @@
+# generic_models
+import msprime
+import stdpopsim.models as models
+
+
+#########
+# Generic Demographic Models
+#
+#
+########
+
+class generic_constant_size(models.Model):
+    """
+    Model Name:
+        generic_constant_size
+
+    Model Description:
+        single-size, single-population model
+
+    Model population indexes:
+        - Population1 : 0
+
+    Parameter Table:
```

Comment:
We can probably just ditch the Parameter Table section here, probably won't be relevant.